requirements.txt

```diff
@@ -5,7 +5,7 @@ canonicalwebteam.discourse==5.4.2
 canonicalwebteam.image-template==1.3.1
 canonicalwebteam.store-api==4.1.0
 canonicalwebteam.docstring-extractor==1.2.0
-canonicalwebteam.store-base==0.0.2
+canonicalwebteam.store-base==0.0.1
```

Comment:
is this right? Downgrading from 0.0.2 -> 0.0.1

Reply:
Yes, it was upgraded automatically, changes in 0.0.2 are breaking and I only want to upgrade when we merge the branch for making beta-app the main app into the main branch.